optimum/onnxruntime/quantization.py

```diff
@@ -351,8 +355,13 @@ def quantize(
                 has_subgraphs = True
                 break
 
-        if quantization_config.is_static and has_subgraphs:
-            raise NotImplementedError("Static quantization is currently not supported for models with" " subgraphs.")
+        if has_subgraphs:
+            if quantization_config.is_static:
+                raise NotImplementedError("Static quantization is currently not supported for models with subgraphs.")
+            if parse(ort_version) >= Version("1.16.0"):
```

Comment:
```suggestion
            if parse(ort_version) == Version("1.16.0"):
```

Reply:
`>=` feels more safe in case this wasn't fixed in the next release but no strong opinion on this